### DISCUSSION TITLE
sstp-client for M1

### DISCUSSION THREAD

**Discussion by snowcube-ui** [Homebrew/discussions#957] [Everyday usage]
Hey,

trying to downgrade sstp-client since 1.0.14 is broken and I have M1.

wget https://homebrew.bintray.com/bottles/sstp-client-1.0.13.catalina.bottle.tar.gz

brew install sstp-client-1.0.13.catalina.bottle.tar.gz 
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from 485f73a32 to 25d098764.
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
crispy-doom                                                                                                             projectm
==> Updated Formulae
Updated 99 formulae.
==> Deleted Formulae
geant4
==> Updated Casks
Updated 41 casks.
==> Deleted Casks
protonmail-unofficial

==> Homebrew was updated to version 3.0.2
The changelog can be found at:
  https://github.com/Homebrew/brew/releases/tag/3.0.2
/opt/homebrew/Library/Homebrew/formulary.rb:55: warning: already initialized constant Formulary::FormulaNamespace688d953d50a6cf293eb3e0e222f6c9c0
/opt/homebrew/Library/Homebrew/formulary.rb:55: warning: previous definition of FormulaNamespace688d953d50a6cf293eb3e0e222f6c9c0 was here
Warning: sstp-client 1.0.14 is available and more recent than version 1.0.13.
==> Caveats
sstpc reads PPP configuration options from /etc/ppp/options. If this file
does not exist yet, type the following command to create it:

sudo touch /etc/ppp/options
==> Summary
🍺  /opt/homebrew/Cellar/sstp-client/1.0.13: 17 files, 179.2KB

$ /opt/homebrew/opt/sstp-client/sbin/sstpc 
dyld: Library not loaded: /opt/homebrew/opt/libevent/lib/libevent-2.1.7.dylib
  Referenced from: /opt/homebrew/opt/sstp-client/sbin/sstpc
  Reason: no suitable image found.  Did find:
	/opt/homebrew/opt/libevent/lib/libevent-2.1.7.dylib: mach-o, but wrong architecture
	/opt/homebrew/opt/libevent/lib/libevent-2.1.7.dylib: stat() failed with errno=1
	/opt/homebrew/Cellar/libevent/2.1.12/lib/libevent-2.1.7.dylib: mach-o, but wrong architecture
Abort trap: 6
$

do you know where i could find sstp-client for M1 in version 1.0.13 ?
thx

**Comment by carlocab** [comment]
You can open a pull request to downgrade `sstp-client` to 1.0.13. You'll also need to add `version_scheme 1` right after the `license` line to avoid `brew audit` complaining about version numbers decreasing.

Here's a guide to help you get started: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request
